src/scss/grommet-core/_objects.form-field.scss

```diff
@@ -92,6 +92,12 @@ $colored-select-drop-caret: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAB
       padding: 0 $form-horizontal-padding;
     }
 
+    &.#{$grommet-namespace}search-input__input {
+      // adjust padding for search-input so input doesn't
+      // overlay search icon
+      padding-right: round($inuit-base-spacing-unit * 2);
```

Comment:
Are we moving the icon as well when in a FormField? Or, why isn't this needed in the non-FormField context?